discord/momobot.py

```diff
@@ -71,6 +72,21 @@ def getFilenameFromUrl(url):
     return url[url.rindex('/')+1:]
 
 
+def toraList():
+  tora_home = "http://home.runtimeexception.net/photos/tora/"
+  tora_data = urllib.request.urlopen(tora_home)
+  tora_lines = tora_data.readlines()
+  torasUrls = []
+
+  for x in tora_lines:
+    s = str(x, 'utf-8')
+    r = re.findall("\"\w+\-?\w*-min.jpg\"", s)
```

Comment:
this regex only works if the naming of pictures remains consistent as current displayed, for example it would not work for a-b-c-d-e-f.jpg

More realistically, if you take pictures with a different phone it could follow a different naming scheme altogether, such as december-1-2015-image-2315.jpg, or maybe .png or something

One way to increase reliability is to make a regex with 3 parenthesized sections,`"(r1)(r2)(r3)`. where `r1` matches `<a href="`, `r2` matches literally anything, and `r3` matches `">`. 

Then, using `re.match` you can index into the returned match object to get the string matched by each parenthesized subregex. For example, if you do `match = re.match("(r1)(r2)(r3)", inputstr)` then `match[0]` is the string matched by regex `r1`, `match[1]` is the string matched by regex `r2` and `match[2]` is the string matched by regex `r3`

Using this method, you can regex to match on the `href` tag to locate the URL in the webpage while making 0 assumptions about the actual filename

Reply:
Done per discussion; setup for discord-accepted filetypes.